enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -519,20 +529,27 @@ class SubsidyAccessPolicyCreditsAvailableResponseSerializer(SubsidyAccessPolicyR
         help_text='Remaining balance on the entire subsidy, in USD cents.',
     )
     subsidy_expiration_date = serializers.DateTimeField(
-        help_text='',
+        help_text='The date at which the related Subsidy record expires.',
         source='subsidy_expiration_datetime',
     )
     learner_content_assignments = serializers.SerializerMethodField('get_assignments_serializer')
 
+    @extend_schema_field(LearnerContentAssignmentWithContentMetadataResponseSerializer)
     def get_assignments_serializer(self, obj):
         """
         Return serialized assignments if the policy access method is of the 'assigned' type
         """
-        if obj.access_method == AccessMethods.ASSIGNED:
+        if obj.is_assignable:
```

Comment:
nit: may be helpful to fail early when not working with an assignable subsidy access policy, so there's not a dangling `return []` after the several lines of business logic when the subsidy access policy is assignable, somewhat removed from the original conditional.

```python
if not obj.is_assignable:
  return []

# everything else...
```